### PR TITLE
Update RELEASE_NOTES

### PR DESCRIPTION
**Pull request by sils1297**

### CODE REVIEW

**Comment by sils1297** [comment]
CC @underyx @scottbelden @Adrianzatreanu @Makman2 and a lot of other people.

Did I forget anything important?

**Comment by underyx** [comment]
@sils1297 yes, the release name and ASCII art :D

**Comment by sils1297** [comment]
Will be added soon! Needed to change train.

File: RELEASE_NOTES.rst [hunk 1]
```diff
@@ -1,3 +1,75 @@
+coala 0.7.0
+===========
+
+This release, 32 people from all over the world have contributed about 200
```
Comment (fneu) [inline RELEASE_NOTES.rst]: For this release

File: RELEASE_NOTES.rst [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+commits over almost two months.
+
+The focus for this release is certainly on the usability of coala. Usability
```
Comment (fneu) [inline RELEASE_NOTES.rst]: The focus of this release

File: RELEASE_NOTES.rst [hunk 3]
```diff
@@ -1,3 +1,75 @@
...
+testing has shown us some important difficulties, users have to face when
```
Comment (fneu) [inline RELEASE_NOTES.rst]: has made us aware of some important difficulties that users [...]

File: RELEASE_NOTES.rst [hunk 4]
```diff
@@ -1,3 +1,75 @@
...
+trying out coala. We have implemented a lot of measures to lower this barrier.
```
Comment (fneu) [inline RELEASE_NOTES.rst]: implemented some countermeasures

**Comment by fneu** [comment]
ack https://github.com/coala-analyzer/coala/pull/2382/commits/bd616ba602a248ea96b05829c6dbb9365b89e559 https://github.com/coala-analyzer/coala/pull/2382/commits/65bcecb749358a582d19a8000bdd57566f02392c https://github.com/coala-analyzer/coala/pull/2382/commits/6e75b6a0c0d6393f2d6e613049b26bd7c2a6aff9

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**